test/data/skip.py

```diff
@@ -31,3 +33,27 @@ def testwrapper(self, *args, **kwargs):
                 testfunc(self, *args, **kwargs)
         return testwrapper
     return testdecorator
+
+def skipForVersion(version, msg):
```

Comment:
the pytest built in function may just let us have less code to maintain? https://docs.pytest.org/en/7.1.x/how-to/skipping.html

For our purposes:
`@pytest.mark.skipif(sys.version_info < (3, 8), reason="his test doesn't converge with newer psi4 versions >3.8")`

Reply:
Fixed!